examples/redshift-data-warehouse-example/stacks/resources/index_template.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
What is the role of this file? Seems like its inherited from the opensearch example.